css/index.less

```diff
@@ -108,17 +136,14 @@ input {
 }
 @media only screen and (max-width: 1050px) {
 	.question-title {
-		height: 4em;
-		width: 25em;
-	}
-	.answer {
-		height: 4em;
-		width: 20em ;
+		p {
+			font-size: 15px;
```

Comment:
you should be consistent in the units you use. You have px here, and 3 lines down, you're using ems....